src/employees/entities/employees.entity.ts

```diff
@@ -1,17 +1,57 @@
-import { Entity, Column, PrimaryGeneratedColumn, Generated } from 'typeorm';
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  Generated,
+  CreateDateColumn,
+  UpdateDateColumn,
+  OneToMany,
+} from 'typeorm';
+import { Employee_Saving } from './employee_saving.entity';
 
 @Entity()
 export class Employees {
-  @PrimaryGeneratedColumn()
+  @PrimaryGeneratedColumn('uuid')
   @Generated('uuid')
   id_employee: string;
 
   @Column()
   nama: string;
 
   @Column()
-  alamat: string;
+  address: string;
 
   @Column()
-  nomor_telepon: string;
+  phone_number: string;
+
+  @Column()
+  entry_date: Date;
+
+  @Column()
+  exit_date: Date;
+
+  @Column()
+  total_saving: number;
+
+  @OneToMany(() => Employee_Saving, saving => saving.employee)
+  savings: Employee_Saving[];
```

Comment:
create as lazy. but seems you dont need this because you will get all the saving data which can be so many

Reply:
done